Makefile

```diff
@@ -458,7 +458,7 @@ uninstall:
 # Linters --------------------------------------------- {{{1
 #
 
-PYTHON_SCRIPTS := utils/hare-shutdown utils/hare-status utils/gen-uuid
+PYTHON_SCRIPTS := utils/hare-shutdown utils/hare-status utils/gen-uuid utils/hare-node utils/util.py
```

Comment:
@mssawant We have pcswrap tool (currently hosted in cortx-ha) which is invoked through `hctl` as `hctl node`. This patch seems to remove `hctl node` <-> `pcswrap` connection. Is this really planned?

`hctl node status` is currently used by provisioner (at least in LDR R1) and by CSM (`hctl node shutdown/maintenance`). If we change the semantics of `hctl node`, external teams will raise bugs.

Reply:
No, not planned, just added a comment. I think we can have a sub command for `hctl node` as `hctl node pcs`. Please provide your inputs.

Reply:
Probably we need to get pcswrap back to Hare project and just enlarge its functionality instead of this new script.

Reply:
@gshipra Just for your information this is the [pcswrap tool](https://github.com/Seagate/cortx-ha/tree/main/ha/pcswrap) we're talking about. Before switching to Github, it was developed by us in Hare repository. And we still use it (you've already seen it in utils/hare-node script).

Reply:
I think we need to check with cortx-ha team because they are now managing pacemaker. So probably its a good idea to transfer the ownership of pcswrap to cortx-ha team.

Reply:
I need to check more about this.

Reply:
I have added code for hctl node --pcs. Please check and give me your feedback.